Tests/RequestBuilderTests.swift

```diff
@@ -24,18 +24,8 @@ class RequestBuilderTests: XCTestCase {
             )]
     }
 
-    func testEndpoint() {
-        let endpoint = RequestBuilder.buildPixelEndpoint()
-        XCTAssert(endpoint != "", "buildPixelEndpoint should return a non-empty string")
-    }
```

Comment:
This test was redundant given the next one performed a more accurate check that implied non-emptyness.